src/recipes/pink-grapefruit-margarita.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "Pink Grapefruit Margarita",
+  "description": "This Pink Grapefruit Margarita is similar to a Paloma but without the soda! This recipe is vegan, gluten-free, and refined sugar-free.",
+  "github": "sherribooher",
+  "ingredients": [
+    "Sea salt, to rim the glass",
+    "2-ounces white tequila",
+    "1-ounce pink grapefruit juice",
+    "1-ounce lime juice",
+    "1-ounce agave",
+    "Slices of grapefruit, for garnish"
+  ],
+  "directions": [
+    "Wet the rim of a glass with a cut piece of the grapefruit or lime",
+    "Pour the sea salt on a plate and dip the wet rim into the salt",
+    "Fill the glass with ice",
+    "Fill a cocktail shaker with ice then add the tequila, pink grapefruit juice, lime juice, and agave",
+    "Shake until cold then strain into the prepared glass",
+    "Garnish with a slice of grapefruit"
+  ],
+  "image": "pink-grapefruit-margarita.jpeg",
```

Looks like a typo with the filetype:
```suggestion
  "image": "pink-grapefruit-margarita.jpg",
```